internal/operators/ocs/validations.go

```diff
@@ -175,7 +176,11 @@ func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID st
 	var countDisks int64
 	var err error
 
-	for _, disk := range disks {
+	eligibleDisks := funk.Filter(disks, func(disk *models.Disk) bool {
+		return disk.InstallationEligibility.Eligible
+	}).([]*models.Disk)
```

Comment:
Why not just add this to the "if" below, to avoid traversing the list of disks an additional time?

Reply:
Done